pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[tool.poetry]
+name = "dochub"
+version = "2.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+Django = "*" 
+celery = "*" 
+pyPdf2 = "*" 
+requests = "*" 
+furl = "*" 
+django-mptt = "*"
+filemagic = "*" 
+beautifulsoup4 = "*" 
+django-compressor = "*"
+whitenoise = "*"
+types-requests = "*"
+django-environ = "*"
+rich = "*" 
+python-slugify = "*"
+types-python-slugify = "*"
+
+[tool.poetry.group.dev.dependencies]
+django-extensions = "*"
+ipython = "*"
```

Comment:
ipython on en a besoin en prod pour débugger

Reply:
on devrait pas a debug en prod, de plus ipython etait deja note comme une dependences de dev dans requirement-in

Reply:
"on devrait pas", je suis d'accord "mais c'est déjà arrivé mille fois, sur dochub ou dans tous mes jobs de devoir le faire et à ce moment là t'es bien content que ./manage.py shell soit un ipython et pas python tout court

Reply:
A discuter pour moi c'est quelque chose a éviter, surtout que on a sentry etc

Reply:
Tu fais comment quand un user a uploadé 200 documents sur la prod qui contiennent du spam et que tu sais pas les virer facilement à la main ? Tu ouvres un shell sur la prod, c'est la seule manière de faire à part couper le serveur, dump la db, la modifier chez toi, la re-upload puis restart le serveur.
Clairement le cas du spam c'est pas encore produit, mais j'ai déjà eu 10x le coup au moins de devoir aller jouer avec ./manage.py shell pour changer la db et c'est un pattern que j'ai vu sur tous les projets web sur lesquels j'ai bossé dans ma vie

Reply:
Tu es censé pouvoir gérer ce genre de cas depuis l'admin, si c'est pas le cas alors faut mettre a jour l'admin. Devoir hotfix des trucs avec ipython en prod ca doit être vraiment la dernière des solutions,